frontend/src/scenes/insights/InsightTabs/InsightTitle.tsx

```diff
@@ -1,24 +1,9 @@
 import React from 'react'
-import { DashboardOutlined } from '@ant-design/icons'
-import { router } from 'kea-router'
-import { useValues } from 'kea'
 
 export function InsightTitle({ actionBar = null }: { actionBar?: JSX.Element | null }): JSX.Element {
-    const {
-        hashParams: { fromItemName, fromDashboard },
-    } = useValues(router)
     return (
         <>
-            <h3 className="l3 insight-title-container">
-                {fromDashboard && (
-                    <DashboardOutlined
-                        style={{ color: 'var(--warning)', marginRight: 4 }}
-                        title="Insight saved on dashboard"
-                    />
-                )}
-                <div className="insight-title-text">{fromItemName || 'Unsaved query'}</div>
-                {actionBar}
-            </h3>
+            <h3 className="l3 insight-title-container">{actionBar}</h3>
```

Comment:
If we're no longer setting the title here this component no longer makes sense, let's get rid of it

Reply:
ok~ Can I get more context on what `annotationsToCreate` is? It's in all of the insight title components but I'm not 100% sure what it's doing